app/src/main/java/com/coveros/coverosmobileapp/blogpost/BlogPostsListActivity.java

```diff
@@ -233,18 +234,33 @@ public void onScroll(AbsListView view, int firstVisibleItem, int visibleItemCoun
          */
         @Override
         public void onItemClick(AdapterView<?> parent, View view, int position, long id) {
-            if (position == 0) {
-                startActivity(new Intent(getApplicationContext(), MainActivity.class));
-            } else {
+            String website;
+            String[] stringsToAddLast = {"","#projects","products/secure-ci/","products/selenified/","#services", "services/agile-development/", "services/agile-testing/",
+                    "services/agile-transition/", "services/devops/", "services/software-security/", "services/training/", "#latest-news", "BLOG-LIST", "presentations/",
+                    "publications/", "webinars-videos/",  "#about", "#team", "#contact-us"};
+            Intent newIntent = new Intent(getApplicationContext(), MainActivity.class);
 
-                startActivity(new Intent(getApplicationContext(), BlogPostsListActivity.class));
+            if(isDevSite){
+                //Use dev site URL
+                website = "https://www3.dev.secureci.com/";
```

Comment:
We will eventually want the URLs to be stored as properties; there shouldnt be any logic for determining which url to use